code/modules/mob/living/simple_animal/hostile/terror_spiders/verbs.dm

```diff
@@ -0,0 +1,160 @@
+		return prob(20)
+	return ..()
+
+
+
+// ---------- WRAP
+
+/mob/living/simple_animal/hostile/poison/terror_spider/proc/DoWrap()
+	if(!cocoon_target)
+		var/list/choices = list()
+		for(var/mob/living/L in view(1,src))
+			if(L == src)
+				continue
+			if(Adjacent(L))
+				if(L.stat != CONSCIOUS)
+					choices += L
+		for(var/obj/O in loc)
+			if(Adjacent(O))
+				if(istype(O, /obj/effect/spider/terrorweb))
+				else
+					choices += O
+		cocoon_target = input(src,"What do you wish to cocoon?") in null|choices
+	if(cocoon_target && busy != SPINNING_COCOON)
+		busy = SPINNING_COCOON
+		visible_message("<span class='notice'>\the [src] begins to secrete a sticky substance around \the [cocoon_target].</span>")
+		stop_automated_movement = 1
+		walk(src,0)
+		spawn(20)
+			if(busy == SPINNING_COCOON)
+				if(cocoon_target && istype(cocoon_target.loc, /turf) && get_dist(src,cocoon_target) <= 1)
+					var/obj/effect/spider/cocoon/C = new(cocoon_target.loc)
+					var/large_cocoon = 0
+					C.pixel_x = cocoon_target.pixel_x
+					C.pixel_y = cocoon_target.pixel_y
+					for(var/obj/item/I in C.loc)
+						I.loc = C
+					for(var/obj/structure/S in C.loc)
+						if(!S.anchored)
+							S.loc = C
+							large_cocoon = 1
+					for(var/obj/machinery/M in C.loc)
+						if(!M.anchored)
+							M.loc = C
+							large_cocoon = 1
+					for(var/mob/living/L in C.loc)
+						if(istype(L, /mob/living/simple_animal/hostile/poison/terror_spider))
+							continue
+						large_cocoon = 1
+						fed++
+						last_cocoon_object = 0
+						L.loc = C
+						C.pixel_x = L.pixel_x
+						C.pixel_y = L.pixel_y
+						visible_message("<span class='danger'>\the [src] sticks a proboscis into \the [L] and sucks a viscous substance out.</span>")
+						break
+					if(large_cocoon)
+						C.icon_state = pick("cocoon_large1","cocoon_large2","cocoon_large3")
+			cocoon_target = null
+			busy = 0
+			stop_automated_movement = 0
+
+
+
+/mob/living/simple_animal/hostile/poison/terror_spider/verb/EatCorpse()
+	set name = "Eat Corpse"
+	set category = "Spider"
+	set desc = "Takes a bite out of a humanoid. Increases regeneration. Use on dead bodies is preferable!"
+	var/choices = list()
+	for(var/mob/living/L in view(1,src))
+		if(L == src)
+			continue
+		if(L in nibbled)
+			continue
+		if(Adjacent(L))
+			if(L.stat != CONSCIOUS)
+				choices += L
+	var/nibbletarget = input(src,"What do you wish to nibble?") in null|choices
+	if(!nibbletarget)
+		// cancel
+	else if(!isliving(nibbletarget))
+		to_chat(src, "[nibbletarget] is not edible.")
+	else if(nibbletarget in nibbled)
+		to_chat(src, "You have already eaten some of [nibbletarget]. Their blood is no use to you now.")
+	else
+		nibbled += nibbletarget
+		regen_points += regen_points_per_kill
+		to_chat(src, "You take a bite out of [nibbletarget], boosting your regeneration for awhile.")
+		src.do_attack_animation(nibbletarget)
+		if(spider_debug)
+			to_chat(src, "You now have " + num2text(regen_points) + " regeneration points.")
```

Comment:
You probably wanna do `"You now have [regen_points] regeneration points."` instead - or is there some reason you're using `num2text`?